content/en/users/aai/check-in/faq/_index.md

```diff
@@ -9,6 +9,19 @@ description: >
   Most frequent questions about EGI Check-in
 ---
 
+## Getting help
+
+Requests for technical assistance and other issues related to Check-in can be
+obtained by sending an e-mail to:
+
+**checkin-support@mailman.egi.eu**
+
+Alternatively, if you already have a Check-in account, the [EGI HelpDesk](https://ggus.eu/)
```

Comment:
Here you may instead link to https://docs.egi.eu/internal/helpdesk/user-guide/ (using the relative path), it provides some info on how to use the helpdesk

Reply:
OK, I will add it. I though about adding documentation but this section in my opinion is not really necessary (only the email contact) so I tried to make it brief.

Reply:
Done.  As I said, users normally should use the email channel, which uses GGUS underneath. The section about GGUS does not add much in practice, so it was more for users knowing GGUS already rather than people starting to use GGUS - but it never hurts to add more information.